core/templates/dev/head/pages/base.html

```diff
@@ -84,6 +84,7 @@
           '{{can_create_collections|js_string}}'),
         csrf_token: JSON.parse('{{csrf_token|js_string}}'),
         DEV_MODE: JSON.parse('{{DEV_MODE|js_string}}'),
+        errorCode: JSON.parse('{{code}}' | '0'),
```

Comment:
nice! I was trying '200'. I guess the issue was when no error was generated ("code" key wasn't present) it didn't knew what to assign to errorCode, right ?

Reply:
@shubha1593 Absolutely correct. That's why I have added an extra or so that when no error code is generated ```errorCode``` will get the value of 0.